src/components/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,109 @@
+import { FC, useState, useEffect } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import { Person } from '../../types';
+import { Loader } from '../Loader';
+import { getPeople } from "../../api";
+
+export const PeoplePage: FC = () => {
+  const [peoples, setPeoples] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const { personSlug } = useParams();
+
+  const personInfo = ['Name', 'Sex', 'Born', 'Died', 'Mother', 'Father'];
+
+  const personName = (name: string | null) => {
+    return peoples.find(person => person.name === name);
+  };
+
+  useEffect(() => {
+    setLoading(true);
+
+    getPeople()
+      .then(setPeoples)
+      .catch(() => setError(true))
+      .finally(() => setLoading(false));
+  }, []);
```

Comment:
The dependency array of this `useEffect` is empty, which means it will run once when the component mounts. If the `getPeople` function depends on some props or state, you should include them in the dependency array.